misk-embedded/README.md

```diff
@@ -0,0 +1,66 @@
+Misk Embedded
+=============
+
+Sometimes we want to embed a Misk Service in another JVM project (such as another Misk service,
+but not necessarily) while avoiding classpath collisions on dependencies. For example, some services
+may use Hibernate 4 and want to embed Misk applications that use Hibernate 5.
+
+Misk embedded encapsulates a Misk service and its classpath for use in another service.
+
+
+Requirements
+------------
+
+The embedded project should have a client/ and service/ submodule: 
+ * The service/ submodule should depend on the client/ submodule
+ * The client/ submodule MUST NOT depend on the service/ submodule.
+
+The consuming project's depends on the embedded project's client/ submodule.
+
+
+Usage
+-----
+
+1. Create an embedding interface in your service's client/ submodule.  
+
+2. Implement that embedding interface in your service's service/ submodule.  
+
+3. Implement an `InjectorFactory` in your service's service/ submodule. The InjectorFactory class
```

Comment:
One more constraint here . . . the InjectorFactory must bind the embedding interface.

Reply:
(please add a note on the docs!)